### PR TITLE
Add --wait cli option for the test command

### PR DESCRIPTION
**Pull request by kot-lex**
Hello there, 

In my tests I quite often run into an issue when screenshot is being taken before example is fully rendered. So I've added wait option to the styleguidist-visual test command which could help to solve such issues.

Here is an example of incorrect screenshots
Button_1_desktop.new.png
button_1_desktop new
Button_1_desktop.png
button_1_desktop

### CODE REVIEW

**Comment by kot-lex** [comment]
I guess this could be a solution for #17 as well

**Comment by kot-lex** [comment]
Current delay functionality forces me to define custom action for every component.
That means I'll have to copy the same code over all example files in our styleguide. It doesn't seem like elegant solution to me.
This PR provides an easy way to set global delay before each screenshot without source code modifications while #22 is more for fine tuning specific components.
I believe those two methods complement each other quite well.

**Comment by unindented** [comment]
Yeah, I agree.

File: src/utils/page.js [hunk 1]
```diff
@@ -78,8 +78,13 @@ async function takeNewScreenshotsOfPreviews (page, previewMap, { dir, progress,
   }
 }
 
-async function takeNewScreenshotOfPreview (page, preview, index, actionState, { dir }) {
+async function takeNewScreenshotOfPreview (page, preview, index, actionState, { dir, delay: wait }) {
```
Comment (unindented) [inline src/utils/page.js]: Why `delay: wait`? Shouldn't it just be `wait`?

Comment (kot-lex) [reply src/utils/page.js]: Sure, sorry for that. I named option "delay" at first, but then renamed it to "wait" to make it consistent with the wait option in custom actions.

Comment (kot-lex) [reply src/utils/page.js]: Just pushed a fix

**Comment by unindented** [comment]
Awesome, thank you!

**Comment by unindented** [comment]
@kot-lex Just published `v0.7.0` with your changes!

**Pull request merged**